packages/icons/src/Icon.jsx

```diff
@@ -1,9 +1,12 @@
+import PropTypes from 'prop-types'
 import React from 'react'
 import styled from 'styled-components'
-import upperFirst from 'lodash.upperfirst'
-import PropTypes from 'prop-types'
 import * as icons from './index'
 
+function upperFirst(string) {
```

Comment:
(n-b) You've written this function twice -- the other instance is in `utils`
It's so small that repetition is probably fine, but do you want to consolidate?